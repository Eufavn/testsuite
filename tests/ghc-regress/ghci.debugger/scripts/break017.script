:l ../QSort.hs
:set -fbreak-on-exception
:trace qsort ("abc" ++ undefined)
:back
:print as
:force as
-- this should print the exception
:print as
