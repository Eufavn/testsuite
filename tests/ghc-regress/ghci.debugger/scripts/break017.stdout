"Stopped at <exception thrown>
_exception :: e
Logged breakpoint at ../QSort.hs:6:23-37
_result :: [Char]
a :: Char
as :: [Char]
as = 'b' : 'c' : (_t1::[Char])
*** Exception: Prelude.undefined
as = 'b' : 'c' : (_t2::[Char])
